src/app/widgets/widget-racer-line/widget-racer-line.component.ts

```diff
@@ -0,0 +1,365 @@
+      case 'contrast':
+        this.labelColor = this.theme().contrastDim;
+        this.valueColor = this.theme().contrast;
+        break;
+      case 'blue':
+        this.labelColor = this.theme().blueDim;
+        this.valueColor = this.theme().blue;
+        break;
+      case 'green':
+        this.labelColor = this.theme().greenDim;
+        this.valueColor = this.theme().green;
+        break;
+      case 'pink':
+        this.labelColor = this.theme().pinkDim;
+        this.valueColor = this.theme().pink;
+        break;
+      case 'orange':
+        this.labelColor = this.theme().orangeDim;
+        this.valueColor = this.theme().orange;
+        break;
+      case 'purple':
+        this.labelColor = this.theme().purpleDim;
+        this.valueColor = this.theme().purple;
+        break;
+      case 'grey':
+        this.labelColor = this.theme().greyDim;
+        this.valueColor = this.theme().grey;
+        break;
+      case 'yellow':
+        this.labelColor = this.theme().yellowDim;
+        this.valueColor = this.theme().yellow;
+        break;
+      default:
+        this.labelColor = this.theme().contrastDim;
+        this.valueColor = this.theme().contrast;
+        break;
+    }
+    this.dtsColor = this.valueColor;
+  }
+
+  ngOnDestroy() {
+    console.log('ngOnDestroy!');
+    this.isDestroyed = true;
+    this.destroyDataStreams();
+    this.canvasService.clearCanvas(this.dToLineContext, this.dToLineElement.width, this.dToLineElement.height);
+
+    if (this.skRequestSubscription !== null) {
+      this.skRequestSubscription.unsubscribe();
+      this.skRequestSubscription = null;
+    }
+  }
+
+  private updateCanvas(): void {
+    this.drawDToLine();
+    this.drawLenBias();
+    this.drawUnit();
+  }
+
+  private drawDToLine(): void {
+    if (this.dToLineCanvas) {
+      this.canvasService.clearCanvas(this.dToLineContext, this.dToLineElement.width, this.dToLineElement.height);
+      const valueText = this.getValueText();
+      this.canvasService.drawText(
+        this.dToLineContext,
+        valueText,
+        Math.floor(this.dToLineElement.width / 2),
+        Math.floor((this.dToLineElement.height / 2) * 1.15),
+        this.maxValueTextWidth,
+        this.maxValueTextHeight,
+        'bold',
+        this.dtsColor
+      );
+    }
+  }
+
+  private getValueText(): string {
+    if (this.dtsValue === null) {
+      return '--';
+    }
+    return this.dtsValue.toFixed(this.widgetProperties.config.numDecimal);
+  }
+
+  private drawUnit(): void {
+    const unit = this.widgetProperties.config.paths['dtsPath'].convertUnitTo;
+    this.canvasService.drawText(
+      this.dToLineContext,
+      unit,
+      Math.floor(this.dToLineElement.width - 10 * this.canvasService.scaleFactor),
+      Math.floor(this.dToLineElement.height - 10 * this.canvasService.scaleFactor),
+      Math.floor(this.dToLineElement.width * 0.25),
+      Math.floor(this.dToLineElement.height * 0.15),
+      'bold',
+      this.valueColor,
```

Comment:
Not sure if this is voluntary but I think it should be the same color as the main number? 
```suggestion
      this.dtsColor,
```